inst/model/nonmem/basic/1/1.cpu

```diff
@@ -1 +1 @@
-       3.141
+       2.595
```

Comment:
most of these `inst` changes are just date/time stamps etc. but I reran the test models and refs so that the classes of the summary objects would be updated to include `bbi_model`